File: data/llama3_8B_ins_results/readable_format_HPEC/fortran/file246.f90
subroutine print_array(y, ny)
implicit none

DATA_TYPE, dimension(ny) :: y
integer :: ny
integer :: i
do i = 1, ny
write(0, DATA_PRINTF_MODIFIER) y(i)
if (mod(i - 1, 20) == 0) then
write(0, *)
end if
end do
write(0, *)
end subroutine

File: data/gpt-4o/readable_format_HPEC/cpp/file246.cpp
#include <iostream>
#define DATA_TYPE int
#define DATA_PRINTF_MODIFIER "%d"
using namespace std;
void print_array(DATA_TYPE y[], int ny)
{
    for(int i = 0; i < ny; i++)
    {
        printf(DATA_PRINTF_MODIFIER, y[i]);
        if (i % 20 == 0)
           cout << endl;
    }
    cout << endl;
}
